src/app/shopping-cart/shopping-cart.component.ts

```diff
@@ -0,0 +1,11 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-shopping-cart',
+  templateUrl: './shopping-cart.component.html'
+})
+export class ShoppingCartComponent implements OnInit {
+  constructor() {}
```

Comment:
don't check in things that you are not using. It pollutes the code. Remove the constructor and the implements onInit and well as the ngOnInit()